libs/csrf/csrfprotector.php

```diff
@@ -106,52 +106,52 @@ public static function init($length = null, $action = null)
 
 			/*
 			 * load configuration file and properties
-			 * Check locally for a config.php then check for 
+			 * Check locally for a config.php then check for
 			 * a config/csrf_config.php file in the root folder
 			 * for composer installations
 			 */
 			$standard_config_location = __DIR__ ."/../config.php";
 			$composer_config_location = __DIR__ ."/../../../../../config/csrf_config.php";
 
 			if (file_exists($standard_config_location)) {
-				self::$config = include($standard_config_location);
+				static::$config = include($standard_config_location);
```

Comment:
@thewhit 
That's cool. In fact it'd be amazing if you could also add a test where you inherit the library and perform these actions.

I'm closing this PR for now, could you create an issue for this for further discussion.